### PR TITLE
Updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SOF3**
to adapt to the new repo (pmmp).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,34 +5,39 @@
 You must follow these guidelines if you wish to contribute to the PocketMine-MP code base, or participate in issue tracking.
 
 ## I have a question
-* For questions, please refer to the _#pocketmine_ or _#mcpedevs_ IRC channel on Freenode. There is a [WebIRC](http://webchat.freenode.net?channels=pocketmine,mcpedevs&uio=d4) if you want.
+* For questions, please refer to the _#pmmp_ or _#pocketmine_ IRC channel on Freenode. There is a [WebIRC](http://webchat.freenode.net?channels=pmmp,pocketmine&uio=d4) if you do not want to install an IRC client.
```
Comment (dktapps) [inline CONTRIBUTING.md]: Why remove #mcpedevs?

Comment (SOF3) [inline CONTRIBUTING.md]: I don't understand why you want to ask PocketMine questions on `#mcpedevs`.

Comment (dktapps) [inline CONTRIBUTING.md]: It doesn't say _PocketMine_ questions, only "I have a question". This might be a technical question related to the game itself.

Comment (SOF3) [inline CONTRIBUTING.md]: I believe the link to `#mcpedevs` served more like a hidden advertisement. I really don't think anyone who wanted to create an issue on PocketMine-MP should actually be redirected to `#mcpedevs`, but would only make more people talking at the wrong channel.

Comment (dktapps) [inline CONTRIBUTING.md]: 👍 point taken. Maybe a small notice to direct non-PocketMine-related questions to #mcpedevs? Up to you.

**Review by dktapps — APPROVED** [review]

**Comment by archie426** [comment]
I think this is fine, go for it and merge

**Pull request merged**